tests/components/group/test_sensor.py

```diff
@@ -812,3 +813,197 @@ async def test_sensors_attributes_added_when_entity_info_available(
     assert state.attributes.get(ATTR_ICON) is None
     assert state.attributes.get(ATTR_STATE_CLASS) == SensorStateClass.TOTAL
     assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == "L"
+
+
+async def test_sensor_state_class_no_uom_not_available(
+    hass: HomeAssistant,
+    caplog: pytest.LogCaptureFixture,
+) -> None:
+    """Test the sensor losing its UoM while having a state class it goes unavailable."""
```

Comment:
I find this docstring hard to understand, can it be approved?

Reply:
I shortened it and added a few comment lines to clarify.